addons/more-costume-fonts/userscript.js

```diff
@@ -0,0 +1,63 @@
+export default async function ({ addon, console }) {
+	
+	const paper = await addon.tab.traps.getPaper();
+	const updateImage = paper.tool.onUpdateImage.bind('paper.tool');
+	const fonts = [
+		{ 'name': 'Sans Serif', 'family': 'Sans Serif' },
+		{ 'name': 'Serif', 'family': 'Serif' },
+		{ 'name': 'Handwriting', 'family': 'Handwriting' },
+		{ 'name': 'Marker', 'family': 'Marker' },
+		{ 'name': 'Curly', 'family': 'Curly' },
+		{ 'name': 'Pixel', 'family': 'Pixel' },
+		{ 'name': '中文', 'family': '"Microsoft YaHei", "微软雅黑", STXihei, "华文细黑"' },
+		{ 'name': '日本語', 'family': '"ヒラギノ角ゴ Pro W3", "Hiragino Kaku Gothic Pro", Osaka, "メイリオ", Meiryo, "ＭＳ Ｐゴシック", "MS PGothic"' },
+		{ 'name': '한국어', 'family': 'Malgun Gothic' },
+		{ 'name': 'Scratch', 'family': 'Scratch' },
+		{ 'name': 'Arial', 'family': 'Arial' },
+		{ 'name': 'Helvetica', 'family': 'Helvetica' },
+		{ 'name': 'Verdana', 'family': 'Verdana' },
+		{ 'name': 'Impact', 'family': 'Impact' },
+		{ 'name': 'Monospace', 'family': 'Monospace' }
+	]
+	
+	function setFont(font, update) {
+		paper.tool.changeFont(font.font);
+		document.querySelector("[class*='font-dropdown_mod-unselect']").click();
+		updateImage();
+	}
+	
+	function addNewFont(font, previousFont, modContextMenu) {

```

Comment:
Can you just add a check for `if (!addon.self.disabled) return` at the top of this function and then add `dynamicEnable` and `dynamicDisable` in the addon.json, so the addon can be turned on and off dynamically?